juriscraper/opinions/united_states/state/conn.py

```diff
@@ -73,13 +85,159 @@ def _process_html(self) -> None:
         """
         for row in self.html.xpath(".//*[contains(@href, '.pdf')]"):
             pub = row.xpath('preceding::*[contains(., "Published")][1]/text()')
-            date = self.find_published_date(pub[0])
+            if pub:
+                date_filed_is_approximate = False
+                date_filed = self.find_published_date(pub[0])
+            else:
+                # May happen on most recent opinions, which have a header like
+                # "Publication in the Connecticut Law Journal To Be Determined"
+                date_filed = f"{self.current_year}-07-01"
+                date_filed_is_approximate = True
+
```

Comment:
Seems OK to me. I assume that we'd want to get these via a second scraper or a later pass?

Reply:
They will be ingested by this scraper, the `date_filed` is available inside the documents, so we try to pick it via `extract_from_text`

For example, for the first row in this year's [page](https://www.jud.ct.gov/external/supapp/archiveAROsup24.htm)


![image](https://github.com/user-attachments/assets/750f2c18-16b6-44c4-9243-a05b6fa1743b)